pkg/backup/backup.go

```diff
@@ -181,9 +200,111 @@ func (kb *kubernetesBackupper) Backup(backup *api.Backup, data io.Writer) error
 		}
 	}
 
+	if err := kb.backupAdditionalItems(ctx); err != nil {
+		errs = append(errs, err)
+	}
+
+	return kuberrs.NewAggregate(errs)
+}
+
+func (kb *kubernetesBackupper) backupAdditionalItems(ctx *backupContext) error {
+	var (
+		errs          []error
+		resolvedItems = make(map[itemKey]schema.GroupResource)
+	)
+
+	// resolve additionalItems to GroupResources
+	for _, itm := range ctx.additionalItems {
+		groupResource, err := resolveGroupResource(kb.discoveryHelper.Mapper(), itm.resource)
```

Comment:
If we modify `resolveGroupResource` to be `resolveGroupVersionResource` and return a `schema.GroupVersionResource`, we'll have the version we need, and you won't need the new `clientForGroupResource` function below. Let me know if you need further clarification.

Reply:
We still need the APIResource obj to be able to get a client, how would you do that differently?